testing/ostest/ostest_main.c

```diff
@@ -295,7 +321,9 @@ static int user_main(int argc, char *argv[])
   show_environment(false, true, true);
   check_test_memory_usage();
 
+#ifndef CONFIG_BUILD_KERNEL
```

Comment:
why skip

Reply:
clearenv() will clear PATH, result in failure to find elf binary for sub-task spawn